### DISCUSSION TITLE
Windows Service

### DISCUSSION THREAD

**Discussion by brunobaraobr** [bluenviron/mediamtx#807] [Questions] [answered]
Hi,

I'm trying to create a windows service with the rtsp simple server, but it returns an error. If I run it directly, it works, but when trying to create a service it returns an error.

Anyone with the same problem or any tips?

Thank you so much guys.

**Comment by Gauthamraju31** [comment]
Hi @brunobaraobr, Can you provide a brief explanation of how you are creating a windows service?

**Comment by aler9** [comment, marked as answer]
Instructions on how to use rtsp-simple-server as a Windows service have been added to the README:

https://github.com/aler9/rtsp-simple-server#windows

**Reply by brunobaraobr** [reply]
Thanks!!!

[a comment by an automated account is not shown]
